modules/e2ee/Worker.js

```diff
@@ -1,378 +1,382 @@
-// Worker for E2EE/Insertable streams. Currently served as an inline blob.
-const code = `
-    // Polyfill RTCEncoded(Audio|Video)Frame.getMetadata() (not available in M83, available M84+).
-    // The polyfill can not be done on the prototype since its not exposed in workers. Instead,
-    // it is done as another transformation to keep it separate.
-    function polyFillEncodedFrameMetadata(encodedFrame, controller) {
-      if (!encodedFrame.getMetadata) {
+/* global TransformStream */
+
+// Worker for E2EE/Insertable streams.
+//
+
+/**
+ * Polyfill RTCEncoded(Audio|Video)Frame.getMetadata() (not available in M83, available M84+).
+ * The polyfill can not be done on the prototype since its not exposed in workers. Instead,
+ * it is done as another transformation to keep it separate.
+ */
+function polyFillEncodedFrameMetadata(encodedFrame, controller) {
+    if (!encodedFrame.getMetadata) {
         encodedFrame.getMetadata = function() {
-          return {
-            // TODO: provide a more complete polyfill based on additionalData for video.
-            synchronizationSource: this.synchronizationSource,
-            contributingSources: this.contributingSources
-          };
+            return {
```

Comment:
we should remove this and drop M83 support. Later

Reply:
Agreed. Since it's minimal, and I don't want to add a version check, I thought we can keep it around for a bit longer.